templates/taxi/car_list.html

```diff
@@ -2,15 +2,50 @@
 
 {% block content %}
   <h1>Car list</h1>
+  <h1>Car list
+    <a style="float: right" href="{% url "taxi:car-create" %}">+</a>
+  </h1>
```

Comment:
There is a duplicate `<h1>` tag for 'Car list'. Consider removing one of them to avoid redundancy.